openedx/core/djangoapps/discussions/models.py

```diff
@@ -89,20 +89,34 @@ def pii_sharing_required_message(provider_name):
 
 AVAILABLE_PROVIDER_MAP = {
     'legacy': {
-        'features': [
-            Features.ANONYMOUS_POSTING.value,
-            Features.BLACKOUT_DISCUSSION_DATES.value,
-            Features.COMMUNITY_TA_SUPPORT.value,
-            Features.COURSE_COHORT_SUPPORT.value,
-            Features.DISCUSSION_PAGE.value,
-            Features.INTERNATIONALIZATION_SUPPORT.value,
-            Features.PRIMARY_DISCUSSION_APP_EXPERIENCE.value,
-            Features.QUESTION_DISCUSSION_SUPPORT.value,
-            Features.RESEARCH_DATA_EVENTS.value,
-            Features.REPORT_FLAG_CONTENT_TO_MODERATORS.value,
-            Features.WCAG_2_0_SUPPORT.value,
-            Features.WCAG_2_1.value,
-        ],
+        'features': {
+            'basic_support': [
+                Features.DISCUSSION_PAGE.value, # check where this goes
+                Features.LTI_BASIC_CONFIGURATION.value,
+                Features.PRIMARY_DISCUSSION_APP_EXPERIENCE.value,
+            ],
+            'partial_support': [
+                Features.ANONYMOUS_POSTING.value,
+                Features.BLACKOUT_DISCUSSION_DATES.value,
+                Features.COMMUNITY_TA_SUPPORT.value,
+                Features.EMAIL_NOTIFICATIONS.value,
+                Features.QUESTION_DISCUSSION_SUPPORT.value,
+                Features.REPORT_FLAG_CONTENT_TO_MODERATORS.value,
+                Features.WCAG_2_0_SUPPORT.value,
+                Features.WCAG_2_1.value, # check where this goes
```

Comment:
@awais-ansari not sure which section within features this goes in! Can you verify where to add `WCAG_2_1`!